uaclient/cli.py

```diff
@@ -43,6 +43,7 @@
 Technical support level: {tech_support_level}
 """
 UA_DASHBOARD_URL = 'https://contracts.canonical.com'
```

Comment:
I think we can drop this constant now, right?

Reply:
dropped. nice catch. sloppy of me.